doc/design/upgrade.md

```diff
@@ -1,20 +1,37 @@
 # Upgrade
 
-After installed VCH, VIC admin would want to upgrade existing instances to latest version. This document describes the upgrade requirement, and design options.
+After installed VCH, VIC admin would want to upgrade existing instances to latest version, to get new features or to fix exposed security issues. This document describes the upgrade requirement, and design options.
```

Comment:
After installing a VCH, a VIC Engine admin will need to manage the lifecycle of the VCH. One aspect of this lifecycle management is upgrading and patching. For our purposes patching and upgrading will be treated the same. 

The VIC Engine product will release a new, complete, software download bundle for both patches and upgrades, versus, having a sparse patch or a separate full upgrade model.